lib/crawly/worker.ex

```diff
@@ -80,11 +76,12 @@ defmodule Crawly.Worker do
     )
 
     case fetcher.fetch(request, options) do
-      {:ok, response} ->
+      {:ok, %HTTPoison.Response{status_code: 200} = response} ->
```

Comment:
Should merge in latest master branch for changes on fetchers

Reply:
I am not quite getting it, but otherwise, I think this is a result of rebase with latest master.

Reply:
because the latest changes in master allows for defining of a custom fetcher, which may not be a `HTTPoson.Response`. Maybe can change it to pattern match on just the map's first-level field `status_code`, or to pattern match on `%Crawly.Response{status_code: 200}`